lmn/tests/test_views.py

```diff
@@ -469,6 +469,24 @@ def test_correct_templates_uses_for_notes(self):
         self.assertTemplateUsed(response, 'lmn/notes/new_note.html')
 
 
+class TestUserNotes(TestCase):
+
+    fixtures = [ 'testing_users', 'testing_artists', 'testing_venues', 'testing_shows', 'testing_notes' ]
+
+    def test_delete_own_note(self):
+        self.client.force_login(User.objects.first())
+        response = self.client.post(reverse('delete_note', args=(1,)), follow=True)
```

Comment:
might not be significant, but response variable is unused in test_delete_own_note so can probably remove line 478 altogether here.